tests/installation/disk_activation/configure_dasd.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Activates multipath when multipath activation message appears
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use parent 'y2_installbase';
+use strict;
+use warnings;
+
+sub run {
+    my $disk_activation = $testapi::distri->get_disk_activation();
+    $disk_activation->configure_dasd_disks();
+    my $dasd_disk_management = $testapi::distri->get_dasd_disk_management();
+    $dasd_disk_management->filter_channel('0.0.0150');
```

Comment:
this is clear a different test, could you please create new test module for this line (i.e: /disk_activation/filter_dasd_device.pm` and for example just schedule once, not in all the yamls.
And for the remaining lines one test to access and another one for finish, which I believe it already exists.

Reply:
yy I had not resolved those from the first time yet!

Reply:
I ll work on this

Reply:
So as we discussed I created a progress ticket for this https://progress.opensuse.org/issues/103320